src/components/WebexMember/WebexMember.jsx

```diff
@@ -22,16 +22,18 @@ export default function WebexMember({
   displayStatus,
 }) {
   const {firstName, lastName} = usePerson(personID);
+  const {ID,orgID} = useMe();
   const members = useMembers(destinationID, destinationType);
   const member = members
     .find((itemMember) => itemMember.personID === personID);
-
   const isMuted = member && member.muted;
-
   return (
     <div className={`${WEBEX_COMPONENTS_CLASS_PREFIX}-member`}>
       <WebexAvatar personID={personID} displayStatus={displayStatus} />
-      <div className="member-name">{`${firstName} ${lastName}`}</div>
+      <div className="member-name">
+        <div>{`${firstName} ${lastName}`}</div>
+        {ID !== personID && destinationType === DestinationType.ROOM && member && member.orgID !== orgID && <div className="member-organization">{`${member.orgID}`}</div>}
```

Comment:
is this feature available only for `DestinationType.ROOM`? Or it can be also for `DestinationType.MEETING`?

Reply:
good question. @lalli-flores, we decided to implement this for meeting, too, as this is how the desktop client behaves.

Reply:
Yeah, works for me